SonarRunner.Shim/Resources.resx

```diff
@@ -148,10 +148,11 @@
     <value>Expected to be called with one argument: the full path to the analysis config file</value>
   </data>
   <data name="ERR_NoProjectInfoFilesFound" xml:space="preserve">
-    <value>No ProjectInfo.xml files were found. Possible causes: 
-1. The project has not been built - the end step was called right after the begin step, without a build step in between 
-2. An unsupported version of MSBuild has been used to build the project. Currently MSBuild 12.0 upwards are supported
-3. The build step has been launched from a different working folder</value>
+    <value>The SonarQube MSBuild integration failed: SonarQube collects mandatory information about your project while it is being built by MSBuild.
```

Comment:
@duncanpMS @bgavrilMS I believe this is a sligthly better, less technical user message.

We don't need to mention `ProjectInfo.xml` at all - that's purely an implementation detail. Some users thought about adding such a file to their project.

Let me know if you have even better suggestions